deployment/environments/production/config.json

```diff
@@ -9,8 +9,9 @@
       "nodePoolCount": 2,
       "minNodeCount": 2,
       "maxNodeCount": 4,
-      "machineType": "n2-standard-2"
+      "machineType": "n2d-standard-2"
```

Comment:
Is n2d supposed to provide better performance?

Reply:
It is roughly the same, but cheaper

Reply:
> This looks good to me. Since you're thinking about this now, you would be an absolute legend if you could also make it possible (and if it's not a huge lift) to choose the instance type when creating a new project (n2, e2, micro etc)...

I'll look into this one, but the list of possible instance types is huge. Before you bootstrap, you can always update the config.json manually :)

Reply:
> but cheaper

Love that part!

> but the list of possible instance types is huge
I don't need an exhaustive list... in fact part of the reason it would be useful is for people who don't know what to choose (which also means they wouldn't know what to put in `config.json`)... ie something this would be ideal (as a CLI select option):

- `e2-micro` - Test project or demo, optimize for cost.
- `n2d-standard-2` - Standard baseline with room for growth (default).
-  `???????` - Massive project. Optimize for performance.
-  `other` - (Type in own...potentially will fail if incorrect or unknown but that's OK).